### ISSUE TITLE
Bazel 7 + MODULE

### ISSUE THREAD

**Issue by chulkilee** [njlr/rules_pycross_oci#2]
I confirmed it works well with Bazel 6 + WORKSPACE (#1) but I failed to migrate it to Bazel 7 + MODULE...

**Comment by njlr** [comment]
Perhaps open an issue on the relevant `rules_` repo?

**Comment by chulkilee** [comment]
^ created #3 

Also.. I'm thinking of adding this to other examples repo - probably https://github.com/aspect-build/bazel-examples/tree/main/oci_python_image - is it okay to bring the example in this repo with my change there? It would help other people to find out the example.

**Comment by njlr** [comment]
Yes please feel free to copy it anywhere - I mostly made this one for my own purposes (much easier to test a small example than in a big monorepo)
